automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/workspace/WorkspaceSpec.scala

```diff
@@ -128,6 +130,100 @@ class WorkspaceSpec extends FreeSpec with WebBrowserSpec with WorkspaceFixtures
           aclEditor.ui.canComputeChecked() shouldBe false
         }
       }
+
+      "should be able to enter workspace attributes" in withWebDriver { implicit driver =>
+        withWorkspace(billingProject, "WorkspaceSpec_add_ws_attrs") { workspaceName =>
+          val listPage = signIn(Config.Users.harry)
+          val detailPage = listPage.openWorkspaceDetails(billingProject, workspaceName).awaitLoaded()
+
+          detailPage.ui.beginEditing()
+          detailPage.ui.addWorkspaceAttribute("a", "a")
+          detailPage.ui.addWorkspaceAttribute("b", "b")
+          detailPage.ui.addWorkspaceAttribute("c", "c")
```

Comment:
here and in other tests: I think you should use different strings for key and value. Otherwise, your test can't test that you haven't swapped key and value somewhere.